app/handlers/baseHandler.py

```diff
@@ -2,31 +2,70 @@
 import json
 import traceback
 import os
+from traffic_cloud_utils.app_config import get_project_path
 
 class BaseHandler(tornado.web.RequestHandler):
     def initialize(self):
         self.error_message = None
         self.MB = 1024*1024
         self.GB = 1024*self.MB
 
-    def find_argument(self, arg_name, default=None):
+    def project_exists(self, identifier):
```

Comment:
Should this instead return a boolean we can store instead of having to call it multiple times, or does it matter?